CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 3.4.4 (Under development)
 
+### AppCenter
+
+#### UWP
+
+* **[Fix]** Fix `Contoso.Forms.UWP.Puppet` crash due to `SQLitePCLRaw.batteries_v2` was not found.
```

Comment:
This is a test app, this does **NOT** affect the SDK. Remove the changes in changelog.

Reply:
Fixed